os-jws-launch/added/launch/http.sh

```diff
@@ -0,0 +1,17 @@
+# bourne shell script snippet
+# used by OpenShift JBoss Web Server launch script
+
+function prepareEnv() {
+  unset JWS_SERVER_NAME
+}
+
+function configure() {
+  configure_http
+}
+
+function configure_http() {
+  if [ -n "$JWS_SERVER_NAME" ]; then 
+    sed -i "s|redirectPort=\"8443\"|redirectPort=\"8443\" server=\"${JWS_SERVER_NAME}\"|" $JWS_HOME/conf/server.xml
```

Comment:
I would be much happier if we'd define the JWS_SERVER_NAME operation as "if JWS_SERVER_NAME is specified, it adds the 'server' attribute to all HTTP connectors that didn't already have the attribute set"  and then use a proper XML transformation for it... the current behavior generally assumes no customizations have been done on the server.xml...